openapi/preprocess_spec.py

```diff
@@ -330,20 +330,26 @@ def main():
     )
     args = argparser.parse_args()
 
-    spec_url = 'https://raw.githubusercontent.com/%s/%s/' \
+
+    unprocessed_spec = args.output_spec_path + ".unprocessed"
+    in_spec = ""
+    if os.environ.get("OPENAPI_SKIP_FETCH_SPEC") or False:
```

Comment:
Isn't this the same as:

```python
if os.environ.get("OPENAPI_SKIP_FETCH_SPEC"):
  ...
```

Reply:
newbees to python, but i had an experiment locally:
```
kimmin-mpb:~ zuoxiu.jm$ export OPENAPI_SKIP_FETCH_SPEC=""
kimmin-mpb:~ zuoxiu.jm$ python
Python 2.7.15 (default, Jun 17 2018, 12:46:58) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import os;
>>> if os.environ.get("OPENAPI_SKIP_FETCH_SPEC"):
...     print("yes")
... 
>>> 
```
is it convincing? @brendandburns @roycaihw

Reply:
weird, ok, I'm not a Python expert either...